frontend/src/pages/FaqPage.js

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
+
+import HeaderTypography from "../components/HeaderTypography";
+import ParagraphTypography from "../components/ParagraphTypography";
+import RightArrowWhite from "../assets/img/right-arrow-white.svg";
+import RightArrowGreen from "../assets/img/right-arrow-green.svg";
+import {
+	Box,
+	Collapse,
+	Divider,
+	Table, TableBody,
+	TableCell,
+	TableContainer,
+	TableRow,
+} from "@mui/material";
+import * as React from "react";
+import { activeTextColor, secondaryTextColor, textColor } from "../assets/jss/colorPalette";
+import Grid from "@mui/material/Grid";
+
+export default function FaqPage() {
+
+	return <>
+		<Grid container spacing={2} style={{ marginTop: 20 }}>
+			<Grid item xs={12} sm={12} md={12} lg={8} xl={8}>
+				<HeaderTypography style={{ fontSize: "26px", fontWeight: "bold" }}>FAQ</HeaderTypography>
+				<ParagraphTypography style={{ fontSize: "16px", color: secondaryTextColor }}>Here you can find answers to frequently asked questions.</ParagraphTypography>
+				<Divider style={{ backgroundColor: "rgba(255, 255, 255, 0.65)", width: "35px", height: "2px", marginTop: "12px", marginBottom: "20px" }}/>
+				<FaqTable/>
+			</Grid>
+			<Grid item xs={12} sm={12} md={12} lg={4} xl={4}>
+				<div style={{ backgroundColor: "red", width: "100%", height: "320px" }}>Announcements</div>
```

Comment:
Isn't a ParagraphTypography needed here? Or do other design rules apply here? Also in the next Grid-item.

Reply:
I did not really focus on design in the Admin area, as this is not visible to the actual users of the website.